test/odbc/CMakeLists.txt

```diff
@@ -23,18 +23,15 @@ add_executable(
 
   main
   constants.cpp
+  connection_object.cpp
+  drivers.cpp
   database_objects.cpp
   odbc_handler.cpp
   query_generator.cpp
-  test_connection.cpp
-  test_data_types.cpp
```

Comment:
Also where did most of these tests go

Reply:
I temporarily removed these tests from the CMakeLists.txt for now. They still exist under the `odbc/` directory, they're just not being compiled.

I expect these tests to be organized under the `MSSQL/` subdirectory with this refactor. But for testing purposes, I moved a subset of these tests to the new folder structure (makes it easer to debug & change 4 test files instead of 15 in case I get comments on this refactor). Once the refactor is finalized, I will move the rest of these tests to the new folder structure & put them back in the CMakeLists.txt.